File: examples/python/list_agent_phone_numbers.py
"""
List all phone numbers attached to your Inkbox account.

Usage:
    INKBOX_API_KEY=ApiKey_... python list_agent_phone_numbers.py
"""

import os
from inkbox import Inkbox

inkbox = Inkbox(api_key=os.environ["INKBOX_API_KEY"])

numbers = inkbox.numbers.list()

for n in numbers:
    print(f"{n.number}  type={n.type}  status={n.status}")

File: examples/typescript/list-agent-phone-numbers.ts
/**
 * List all phone numbers attached to your Inkbox account.
 *
 * Usage:
 *   INKBOX_API_KEY=ApiKey_... npx ts-node list-agent-phone-numbers.ts
 */

import { Inkbox } from "../../typescript/src/inkbox.js";

const inkbox = new Inkbox({ apiKey: process.env.INKBOX_API_KEY! });

const numbers = await inkbox.numbers.list();

for (const n of numbers) {
  console.log(`${n.number}  type=${n.type}  status=${n.status}`);
}
